python_examples/remove_uuid_tags/README.md

```diff
@@ -1,52 +1,37 @@
 # Remove Tags Script 
 
 * Local system requirements:
-  - poetry >= 1.3.1 
-  - virtualenv >= 20.16.7 
+  - poetry >= 1.3.1
 
-- This is an example script of how to remove tags from a company using UUID's. 
-- Ensure that removeTags.py is in the same directory as removeTags.xlsx and pyproject.toml
-- All commands should be executed from this directory 
+* This is an example script of how to remove tags from a company using UUID's. 
+* Ensure that removeTags.py is in the same directory as removeTags.xlsx and pyproject.toml
+* All commands should be executed from this directory 
 
 
-*** exporting an authentication token for use in the virtual environment ***
+## Exporting an authentication token for use in the virtual environment
+
 - In your portfolio click in the top right corner and under settings select Manage Access Tokens 
 - Once you generate a token make sure that you view the secret it should look something like this 
-  API-V1 xxxxxxxxxxxxxxxxxxxxxx==.xxxxxxx+xxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxx="
-- using a text editor of your choice create a .auth-token file
-- Inside the .auth-token file take your token secret and add:
-  export CYBERGRX_API_TOKEN="API-V1 xxxxxxxxxxxxxxxxxxxxxx==.xxxxxxx+xxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxx="
+  `API-V1 xxxxxxxxxxxxxxxxxxxxxx==.xxxxxxx+xxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxx="`
```

Comment:
I think the double quote shouldn't be there

```suggestion
  `API-V1 xxxxxxxxxxxxxxxxxxxxxx==.xxxxxxx+xxxxxxxxxxxxxxxxxxxxx/xxxxxxxxxxxxxx=`
```

Reply:
I think you're right. @jsully1720 did you test that these commands work with the double quotes? And do they work *without* the double quotes?

Reply:
That's spot on, that's my fault. It definitely wouldn't work like that